https://bugs.gentoo.org/404979
https://github.com/boto/boto/issues/597
https://github.com/boto/boto/commit/f797759a2df235475cd29e67bddec51e7e8d6336

commit f797759a2df235475cd29e67bddec51e7e8d6336
Author: Mitch Garnaat <mitch@garnaat.com>
Date:   Tue Feb 28 05:52:12 2012 -0800

    Tests need to be in the source distro but shouldn't be installed.  Also, moved some tests scattered throughout the distro to the tests directory.  Fixes #597.

diff --git a/setup.py b/setup.py
index 3ede0a3..cc093da 100644
--- a/setup.py
+++ b/setup.py
@@ -59,11 +59,7 @@ setup(name = "boto",
                   "boto.mturk", "boto.mturk.test", "boto.pyami",
                   "boto.pyami.installers", "boto.pyami.installers.ubuntu",
                   "boto.mashups", "boto.contrib", "boto.manage",
-                  "tests", "tests.autoscale", "tests.cloudfront",
-                  "tests.devpay", "tests.db", "tests.dynamodb",
-                  "tests.ec2", "tests.ec2.cloudwatch", "tests.ec2.elb",
-                  "tests.s3", "tests.sdb", "tests.sqs", "tests.sts",
-                  "tests.utils", "boto.services", "boto.cloudfront",
+                  "boto.services", "boto.cloudfront",
                   "boto.roboto", "boto.rds", "boto.vpc", "boto.fps",
                   "boto.fps.test", "boto.emr", "boto.emr.tests", "boto.sns",
                   "boto.ecs", "boto.iam", "boto.route53", "boto.ses",
